packages/workspace-resolver/src/resolve-workspace.ts

```diff
@@ -153,7 +152,7 @@ export function analyzeWorkspaceDependencies(
   // Exclude the root when analyzing package inter-dependencies
   const packagesWithoutRoot = Array.from(workspacePackages.entries()).filter(
     ([, pkg]) => {
-      return pkg.modular.type !== 'root';
+      return pkg.modular?.type !== 'root';
```

Comment:
Minor thing, but this filter can be inlined to the next `forEach` on line 160 to avoid looping through the workspace twice.

As `type` has been added to the `ModularWorkspacePackage`, this can now be:

```suggestion
      return pkg.type !== 'root';
```